src/routes/backoffice.ts

```diff
@@ -67,6 +73,30 @@ export const triggerBookingsWebhook = async (solarisAccountId) => {
   await triggerWebhook(TransactionWebhookEvent.BOOKING, payload);
 };
 
+export const setAccountIsPKontoFlag = async (personId, isPKonto) => {
+  const person = await getPerson(personId);
+
+  person.account = person.account || {};
+
+  const previousIsPKonto = person.account.isPKonto;
```

Comment:
I guess this one should be `person.account.isPKonto -> person.account.is_pkonto`